[2022-02-09 23:42:30,037] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: spark_first_run.2 2022-02-09T23:37:48.077722+00:00 [queued]>
[2022-02-09 23:42:30,162] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: spark_first_run.2 2022-02-09T23:37:48.077722+00:00 [queued]>
[2022-02-09 23:42:30,162] {taskinstance.py:1087} INFO - 
--------------------------------------------------------------------------------
[2022-02-09 23:42:30,162] {taskinstance.py:1088} INFO - Starting attempt 1 of 4
[2022-02-09 23:42:30,162] {taskinstance.py:1089} INFO - 
--------------------------------------------------------------------------------
[2022-02-09 23:42:30,224] {taskinstance.py:1107} INFO - Executing <Task(PythonOperator): 2> on 2022-02-09T23:37:48.077722+00:00
[2022-02-09 23:42:30,229] {standard_task_runner.py:52} INFO - Started process 179 to run task
[2022-02-09 23:42:30,234] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'spark_first_run', '2', '2022-02-09T23:37:48.077722+00:00', '--job-id', '48', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/task.py', '--cfg-path', '/tmp/tmp9r47t9lj', '--error-file', '/tmp/tmpwv6cj2fp']
[2022-02-09 23:42:30,235] {standard_task_runner.py:77} INFO - Job 48: Subtask 2
[2022-02-09 23:42:30,414] {logging_mixin.py:104} INFO - Running <TaskInstance: spark_first_run.2 2022-02-09T23:37:48.077722+00:00 [running]> on host 430c48696ffb
[2022-02-09 23:42:31,021] {taskinstance.py:1300} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=admin
AIRFLOW_CTX_DAG_ID=spark_first_run
AIRFLOW_CTX_TASK_ID=2
AIRFLOW_CTX_EXECUTION_DATE=2022-02-09T23:37:48.077722+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-02-09T23:37:48.077722+00:00
[2022-02-09 23:42:31,093] {logging_mixin.py:104} INFO - finished
[2022-02-09 23:42:31,093] {python.py:151} INFO - Done. Returned value was: None
[2022-02-09 23:42:31,148] {taskinstance.py:1204} INFO - Marking task as SUCCESS. dag_id=spark_first_run, task_id=2, execution_date=20220209T233748, start_date=20220209T234230, end_date=20220209T234231
[2022-02-09 23:42:31,220] {taskinstance.py:1265} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-02-09 23:42:31,248] {local_task_job.py:151} INFO - Task exited with return code 0
